source/blog/2018-06-15-the-emberjs-times-issue-51.md

```diff
@@ -16,16 +16,22 @@ We have a couple of **fresh 🥒 Ember RFCs** (Request for Comments) in for you
 ## [New release of Ember Twiddle](https://ember-twiddle.com)
 
 Ember Twiddle v0.14.0 was released on Monday June 4. It includes support for Ember 2.18, Babel 6, and new module imports.
-The entire [changelog](https://github.com/ember-cli/ember-twiddle/releases/tag/v0.14.0) shows all the work that has gone into this. 
+The entire [changelog](https://github.com/ember-cli/ember-twiddle/releases/tag/v0.14.0) shows all the work that has gone into this.
 The next version is also being worked on, supporting Ember 3.0 - Ember 3.2.
 You can test that out now at [canary.ember-twiddle.com](https://canary.ember-twiddle.com).
 Ember Twiddle is a playground website for developing small Ember applications backed by Github gists.
 
-## [SECTION TITLE](#section-url)
+## [Is Your Components Not Pointy Enough?](https://github.com/rwjblue/ember-angle-bracket-invocation-polyfill)
+[ember-angle-bracket-invocation-polyfill](https://github.com/rwjblue/ember-angle-bracket-invocation-polyfill) provides a polyfill for angle bracket invocation syntax as described in [RFC 311](https://github.com/emberjs/rfcs/pull/311).  It includes features such as invoking components via angle brackets using TitleCase, self-closing syntax, paths much more.
 
+If you’re interested in using this the best documentation is in the RFC itself.
 ---
 
-## [SECTION TITLE](#section-url)
+## [Back To the Future](https://github.com/rwjblue/ember-named-arguments-polyfill)
+Another polyfill that is now available is [ember-named-arguments-polyfill](https://github.com/rwjblue/ember-named-arguments-polyfill) which polyfills the named arguments feature to work for Ember 2.10+.
```

Comment:
Do you think it would be useful to mention that the named arguments feature has already landed in Ember 3.1 here as well and link to the particular section of the release blog post?
https://www.emberjs.com/blog/2018/04/13/ember-3-1-released.html#toc_named-arguments-1-of-4